frontend/app/components/activity-new.js

```diff
@@ -34,14 +34,31 @@ export default Component.extend(EKMixin, {
     context.sendAction("done", true);
   },
 
+  addMissingMonthWarning(newActivity) {
+    if (
+      !newActivity.monthFrom ||
+      (!newActivity.monthTo && newActivity.yearTo)
+    ) {
+      document.getElementById("missing-month-warning").style.display = "block";
```

Comment:
Use jquery here. I know it probably shows a deprecation or something but we use jquery in other places as well and in that case, rather be consistent than hack around :)